packages/cli/src/commands/account/new.ts

```diff
@@ -8,17 +8,20 @@ import {
 import { privateKeyToAddress } from '@celo/utils/lib/address'
 import { flags } from '@oclif/command'
 import { toChecksumAddress } from 'ethereumjs-util'
+import * as fs from 'fs-extra'
 import { LocalCommand } from '../../base'
 import { printValueMap } from '../../utils/cli'
 
+const ETHEREUM_DERIVATION_PATH = "m/44'/60'/0'/0"
+
 export default class NewAccount extends LocalCommand {
   static description =
     "Creates a new account locally using the Celo Derivation Path (m/44'/52752'/0/0/indexAddress) and print out the key information. Save this information for local transaction signing or import into a Celo node. Ledger: this command has been tested swapping mnemonics with the Ledger successfully (only supports english)"
 
   static flags = {
     ...LocalCommand.flags,
-    password: flags.string({
-      description: 'Choose a password to generate the keys',
+    passwordPath: flags.string({
```

Comment:
I think this should really be `s/password/passphrase` -- this is what it's called in [BIP-39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) and on the Ledger UI